closed/adds/jdk/src/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -189,6 +200,52 @@ public static final boolean isTraceEnabled() {
     private NativeCrypto() {
         //empty
     }
+    public static final boolean isOpenSSLFIPSVersion() {
```

Comment:
Please insert a blank line before this.

Reply:
Actually, I suggest this method is not needed at all - it's only used in this class. Just use `isOpenSSLFIPS` directly.

Reply:
Removed method altogether.